models.py

```diff
@@ -0,0 +1,153 @@
+from datetime import datetime
+from werkzeug.security import generate_password_hash, check_password_hash
+from app import db, login
+from flask_login import LoginManager, UserMixin, login_required
+
+
+# klasa z kontem użytkownika
+class User(UserMixin, db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(64), index=True, unique=True)
+    email = db.Column(db.String(120), unique=True, index=True)
```

Comment:
Again, Check form that the provided email is no longer than 120 characters. If it is inform the user.